pkg/server/server.go

```diff
@@ -44,18 +48,46 @@ func New(conf *config.Configurations) (*BCDBHTTPServer, error) {
 		return nil, err
 	}
 
-	db, err := bcdb.NewDB(conf, lg)
+	var metricsRegistry *prometheus.Registry
+	var metricsServer *http.Server
+	var metricsListen net.Listener
```

Comment:
Lets add a test that checks the endpoint can be used with one or two metrics, like queue size, etc, before moving on